draft-ietf-rtcweb-jsep.xml

```diff
@@ -1784,8 +1784,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           <t>
             <list style="symbols">
 
-              <t>The profile in any "m=" line in any answer MUST
-              exactly match the profile provided in the offer.</t>
+              <t>The profile in any "m=" line in any answer generated 
```

Comment:
The issue with the text "In order to simplify compatibility
          with such endpoints, JSEP implementations MUST follow the
          following rules when processing the media m= sections in an
          offer:"
is that is not clear if it is generating the offer, or processing a received offer to generate an answer. So it might be the quoted sentence that should be clarified by adding "... in a received offer:"?